scipy/stats/tests/test_binned_statistic.py

```diff
@@ -1,5 +1,6 @@
 import numpy as np
 from numpy.testing import assert_allclose
+from pytest import mark
```

Comment:
```suggestion
import pytest
```
For consistency with the rest of stats. Your way saves lines though. Maybe in the future we'll define:
```python3
parametrize = pytest.mark.parametrize
```